src/patternfly/demos/Card/card-content-1.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-right"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
+      {{#> brand brand--attribute='src="/assets/images/pf-logo-small.svg" alt="Logo"'}} 
+      {{/brand}}
+    {{/l-flex}}
+    {{#> dropdown id=(concat card--id "-dropdown-kebab-right-aligned") dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+    {{/dropdown}}
+    {{#> check}} 
+      {{#> check-input check-input--attribute=(concat 'id="' card--id '-check" name="' card--id '-check" aria-labelledby="' card--id '-check-label"')}}{{/check-input}}
+    {{/check}}
+  {{/l-flex}}
+{{/card-header}}
+{{#> card-body}}
+  {{#> content content--modifier="pf-m-no-margin"}}
```

Comment:
Are we adding this just for the purpose of this demo? Or is it a purposeful design decision to offer this? FWIW, you can get all of this spacing using a flex column layout instead. Just use a spacer between the "provided by" and paragraph text.

Reply:
I more so used content to get the sizes and coloring